packages/server/src/organization/organization.controller.ts

```diff
@@ -169,18 +280,65 @@ export class OrganizationController {
     }
     courseInfo.course.timezone = courseDetails.timezone;
 
-    await courseInfo.course
-      .save()
-      .then(() => {
-        return res.status(HttpStatus.OK).send({
-          message: 'Course updated',
+    try {
+      await courseInfo.course.save();
+      //Remove current profs
+      await UserCourseModel.delete({
```

Comment:
Why are we deleting profs? Also we should update usermodel, as it contains an array of course/usercourse model?

Reply:
I deleted the profs because only those profs that are sent in the requests should be profs. The edit form is prepopulated with profs already in course. Why do we need to update usermodel? When I fetch a user that has been changed into a prof, it should automatically be updated right?

Reply:
The model has the following relation:
```
@OneToMany((type) => UserCourseModel, (ucm) => ucm.user)
  @Exclude()
  courses: UserCourseModel[];
```
I think it's not auto-updated once usercourse model rows are removed?

Reply:
I don't get it. What is there to update in the `UserModel`? When we fetch `UserModel.courses` after the `UserCourseModel` has been updated, the `UserModel.courses` will be correct. Otherwise the adding/editing of profs would not work at all with current implementation.